credsweeper/app.py

```diff
@@ -52,7 +52,9 @@ def __init__(self,
                  ml_threshold: Union[float, ThresholdPreset] = ThresholdPreset.medium,
                  find_by_ext: bool = False,
                  depth: int = 0,
-                 size_limit: Optional[str] = None) -> None:
+                 size_limit: Optional[str] = None,
+                 exclude_lines: Optional[List[str]] = None,
+                 exclude_values: Optional[List[str]] = None) -> None:
```

Comment:
if the lines inserted before e.g. find_by_ext - then 55line is kept untouched :)

Reply:
Agree, but it have a chance to brake code that passes arguments by order rather than byname